Makefile

```diff
@@ -19,15 +21,17 @@ clean:
 	- ( cd tools/kfold && $(MAKE) clean )
 	- ( cd tools/serialization && $(MAKE) clean )
 
+.PHONY: all test clean
 test:
-	- ( cd test_suite && $(MAKE) all)
-	- ( ./test_suite/test_all )
+	gcc -fprofile-arcs -ftest-coverage test_suite/test_*.c src/libRecommender.a -lm -o test_suite/test_all.o
+	./test_suite/test_all.o
 
 coverage:
-	- ( echo "Coverage is not supported on Alpine Linux" )
+	gcov test_all.c
+	lcov --capture --directory . --output-file coverage.info
```

Comment:
Is it okay if the line that generates html does not exist here?

`genhtml coverage.info --output-directory out`

Reply:
no need